### DISCUSSION TITLE
Support for IPv6

### DISCUSSION THREAD

**Discussion by SiddharthAgarwal-01** [dotnet/dotNext#157] [Q&A] [answered]
Hi, we are looking to use this repo as a base of our distributed storage system (RAFT implementation). So I wanted to know if this has IPv6 support for DotNext.Net.Cluster module. If not, is there any plan to introduce that in future.

**Comment by sakno** [comment, marked as answer]
Yes, it has IPv6 as well as DNS resolution.

**Reply by SiddharthAgarwal-01** [reply]
Nice. Thanks, anyways we don't want to have any DNS dependency.
